src/Uintra20/Content/uintra20/src/app/feature/project/reusable/ui-elements/dropzone-wrapper/dropzone-wrapper.component.ts

```diff
@@ -9,15 +13,20 @@ import { Component, OnInit, ViewEncapsulation, HostBinding, Input, Output, Event
 export class DropzoneWrapperComponent implements OnInit {
   @Input() customMessage: string;
   @Input() filesLength: number;
+  @Input() maxFiles: number;
   @Output() success = new EventEmitter();
   @Output() removedFile = new EventEmitter();
   @HostBinding('class') className: string;
 
   message: string;
+  config: IDropzoneConfig = {};
 
   constructor() { }
 
   ngOnInit() {
+    if (this.maxFiles) {
+      this.config.maxFiles = this.maxFiles;
```

Comment:
this.config.maxFiles = this.maxFiles || defaultValue

Reply:
+